libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -5,28 +5,43 @@
 
 import pytest
 from langchain_core.documents import Document
-from sqlalchemy import Connection, create_engine, text
-from sqlalchemy.exc import DBAPIError
+from sqlalchemy import create_engine, text
 
 from langchain_community.embeddings import FakeEmbeddings
-from langchain_community.vectorstores.sqlserver import SQLServer_VectorStore
+from langchain_community.vectorstores.sqlserver import (
+    DistanceStrategy,
+    SQLServer_VectorStore,
+)
 
 _CONNECTION_STRING = str(os.environ.get("TEST_AZURESQLSERVER_CONNECTION_STRING"))
 _SCHEMA = "lc_test"
+_COLLATION_DB_NAME = "LangChainCollationTest"
+_TABLE_NAME = "langchain_vector_store_tests"
+_TABLE_DOES_NOT_EXIST = "Table %s.%s does not exist."
+EMBEDDING_LENGTH = 1536
+VECTOR_STORE_TABLE_NAME = "langchain_vector_store_test"
```

Comment:
how is VECTOR_STORE_TABLE_NAME different than _TABLE_NAME

Reply:
Ah, that's an oversight. It can be reused.